src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
```

Comment:
Reporter: FINDBUGS
Rule: MS_SHOULD_BE_FINAL
Severity: INFO
File: se/bjurr/violations/lib/example/OtherClass.java L7
 Source: se.bjurr.violations.lib.example.OtherClass


Field isn't final but should be


   <p>
This static field public but not final, and
could be changed by malicious code or
        by accident from another package.
        The field could be made final to avoid
        this vulnerability.</p>


<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>